### PR TITLE
Add new tracing guide for AWS payload tagging

### PR DESCRIPTION
**Pull request by nevilgeorge**
### What does this PR do? What is the motivation?
Adds a new Tracing Guide for documentation on how to use the new AWS payload tagging feature. 

### Merge instructions

Merge queue is enabled in this repo. To have it automatically merged after it receives the required reviews, create the PR (from a branch that follows the `<yourname>/description` naming convention) and then add the following PR comment:

```
/merge
```

#### Tested locally, here are some screenshots
image
(see last item)

image

image

### CODE REVIEW

**Comment by maycmlee** [comment]
Created [DOCS-9502](https://datadoghq.atlassian.net/browse/DOCS-9502) for docs review.

**Review by brett0000FF — APPROVED** [review]
Thanks for adding this draft! Please see the latest commits for my edited version. Please let me know if there's anything you disagree with or would like to change.

Separately, we should add these config options to the Library Config pages and link to this guide, as suggested.

File: content/en/tracing/guide/aws_payload_tagging.md [hunk 1]
```diff
@@ -0,0 +1,263 @@
...
+    "foo.bar": "baz",  
+    "Arr": ["a", "b"]  
+  }  
+}
+```
+
+Generated tags:
+
+```text
+aws.request.body.Message.foo.\bar: baz
```
Comment (tlhunter) [inline content/en/tracing/guide/aws_payload_tagging.md:52]:
```suggestion
aws.request.body.Message.foo\.bar: baz  
```

**Pull request merged**